### PR TITLE
[EDA] Account Model Recommended Fix Updates

### PR DESCRIPTION
**Pull request by skosgi1**
# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

File: force-app/main/default/labels/CustomLabels.labels-meta.xml [hunk 1]
```diff
@@ -1556,47 +1556,47 @@
         <language>en_US</language>
         <protected>true</protected>
         <shortDescription>Recommended fix when the Administrative Account Record Type is inactive</shortDescription>
-        <value>Verify the Administrative Account Record Type specified in EDA Settings, on the Accounts and Contacts tab. If {0} ({1}) is correct, go to Salesforce Setup and set the record type to Active. If it isn&apos;t correct, change the Administrative Account Record Type to a valid, active record type.</value>
+        <value>In EDA Settings | Account Model, verify the Administrative Account Record Type. If {0} ({1}) is correct, go to Salesforce Setup and set the record type to Active. If it isn't correct, change the Administrative Account Record Type to a valid, active record type.</value>
```
Comment (xinnanlisf) [inline force-app/main/default/labels/CustomLabels.labels-meta.xml:1559]: Should we use apos here too?

Comment (xinnanlisf) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml:1559]: Same applies to other labels.

**Review by xinnanlisf — CHANGES_REQUESTED** [review]
Left some minor comments.

File: force-app/main/default/labels/CustomLabels.labels-meta.xml (hunk 1 above)
Comment (skosgi1) [reply force-app/main/default/labels/CustomLabels.labels-meta.xml]: Thanks Xinnan, fixed these.

**Review by xinnanlisf — APPROVED** [review]

**Pull request merged**